src/pysdmx/model/category.py

```diff
@@ -75,15 +64,12 @@ class CategoryScheme(Struct, frozen=True, omit_defaults=True):
             (e.g. "The SDMX Content Guidelines for Statistical
             Subject-Matter Domains").
         version: The scheme version (e.g. 1.0)
-        categories: The list of top level categories in the scheme.
     """
 
-    id: str
-    name: str
-    agency: str
-    description: Optional[str] = None
-    version: str = "1.0"
-    categories: Sequence[Category] = ()
+    @property
+    def categories(self) -> Sequence[Category]:
+        """Extract the items in the Category Scheme."""
```

Comment:
I would use the old documentation ("The list of top level categories in the scheme."), as it made it explicit that it would be the top level categories only (i.e. no recursive traversal of the categories and their descendants)

Reply:
Adding it to the Docstring of this method.